.github/workflows/install.yml

```diff
@@ -4,15 +4,15 @@ name: Install from scratch
 on:
   schedule:
     # Run at 7am every day
-    # - cron:  '0 7 * * *'
+    - cron:  '0 7 * * *'
     # Run at 7am every Monday
-    - cron:  '0 7 * * 1'
+    # - cron:  '0 7 * * 1'
   workflow_dispatch:
     inputs:
       RESET:
         description: Reset existing platform data?
         type: boolean
-        default: true
+        default: false
```

Comment:
OK this PR works as it is, but this change is not going to do what you expect, as there is a pitfall that you may not have noticed. 

During scheduled workflows, `workflow_dispatch.inputs` are undefined. As a consequence, RESET will be `null` on cron events. Line 26, `inputs.RESET` will be cast as an empty string `''`. Thus, `contains(inputs.RESET, 'true')` will evaluate to `"false"`, and all will be good. But notice that in this process, the `default` value that you've just modified here has no impact whatsoever. That default value is only used when triggering workflows manually.